### DISCUSSION TITLE
[Question] TextArea maxLenght 커스텀 기능

### DISCUSSION THREAD

**Discussion by yangSangHoon** [wanteddev/montage-web#19] [Q&A]
### 요약

TextArea사용중에 maxLength 영역에 커스텀 방법 문의 드립니다.

### 상세

maxLength가 천자리가 넘어가도 콤마를 제거해달라는 요구사항 발생하여 문의 남깁니다.
https://wantedlab.atlassian.net/browse/PI-64323?atlOrigin=eyJpIjoiNDVmY2FiZTI1NmE4NDIzODhkMWE0MWRjNDQ3ZjQ2MDAiLCJwIjoiaiJ9

**Comment by jy-son** [comment]
[디자인시스템](https://www.figma.com/design/7RHtWV3Pw6I98UEDjbx5V1/0-Component?node-id=10930-28442&m=dev) 상 콤마가 붙어있는 형태일 것 같은데요.
입력한 글자 수, 최대 글자 수 표시 형태를 맞추면 좋을 것 같아 보이네요!

**Reply by yangSangHoon** [reply]
넵 그렇게 진행해도 된다고 하네요..
제가 스퀘어프로젝트에 대한 이해도가 낮아서 잘못된 요청을 드렸네요 해당이슈 삭제 하겠습니다
감사합니다!~
